templates/practice/documentation.html

```diff
@@ -0,0 +1,187 @@
+      </div>
+      <div class="p-top u-hide--large"><a href="#" class="p-top__link">Back to top</a></div>
+      <div id="documentation-pillars" style="scroll-margin-top: 4.5rem;">
+        <h2>Four pillars</h2>
+        <p>Our work identifies four pillars that support documentation success</p>
+        <div class="row">
+          <div class="col-4">
+            <h2 class="p-heading--4">Direction</h2>
+            <p>Where we want to go - the standards and quality we seek, so that documentation meets its users’ needs.</p>
+          </div>
+          <div class="col-4">
+            <h2 class="p-heading--4">Care</h2>
+            <p>Our attitude towards documentation, and a culture of documentation discipline that makes it a living concern.</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-4">
+            <h2 class="p-heading--4">Equipment</h2>
+            <p>The tools and machinery we adopt to write, maintain and publish documentation, that serve our work best.</p>
+          </div>
+          <div class="col-4">
+            <h2 class="p-heading--4">Execution</h2>
+            <p>How we do our work, so that we consistently produce better output with less effort and more satisfaction.</p>
+          </div>
+        </div>
+        <p><a href="https://ubuntu.com/blog/the-future-of-documentation-at-canonical">Read about our plan to put these pillars into place at Canonical&nbsp;&rsaquo;</a></p>
+      </div>
+      <div class="p-top u-hide--large"><a href="#" class="p-top__link">Back to top</a></div>
+      <div id="documentation-careers" style="scroll-margin-top: 4.5rem;">
+        <h2>Careers and opportunities</h2>
+        <p>There is a dedicated career progression in documentation at Canonical, including roles for Technical Author, Senior Technical Author and Documentation Manager.</p>
+        <p>Our plans include recruiting a cohort of technical authors to join several engineering teams, working on products across Canonical's portfolio.</p>
+        <p>Documentation is not a siloed activity. You’ll find engineers and product managers making real and substantial contributions, as part of their everyday work.</p>
+        <p>It’s a responsibility that is shared by everyone who works in engineering, product or support roles. We expect everyone to help manage that responsibility.</p>
+        <p>As an engineer, don’t be surprised to be asked about your work in - or thoughts on - documentation at a job interview. As an engineering manager or director, expect to sit an interview dedicated to documentation practice.</p>
+        <div class="p-notification--information">
+          <div class="p-notification__content">
+            <p class="p-notification__message">Don’t hesitate to mention documentation-related work that you're proud of in your application or your interview - it will be valued.</p>
```

Comment:
Same here with the encoder of the dash. It looks better from a typography point of view